test/benchmark_http_client_test.cc

```diff
@@ -121,14 +132,25 @@ class BenchmarkClientHttpTest : public Test {
     decoders_.clear();
     dispatcher_->run(Envoy::Event::Dispatcher::RunType::Block);
     EXPECT_EQ(0, inflight_response_count);
```

Comment:
I'm no expert on this but can you double check that EXPECT_FALSE(true) would fail here, I remember that EXPECT_* don't work in methods and has to be in the main test case.

Reply:
It does fail here.